web_tests/tests/test_login_page.py

```diff
@@ -0,0 +1,17 @@
+def test_valid_login(login_page, valid_user):
+    """Test login with valid credentials."""
+    dashboard_page = login_page.perform_successful_login(valid_user)
+    assert dashboard_page.is_displayed(), "Dashboard page is not displayed"
+
+
+def test_redirecting_to_orangehrm(login_page):
+    """Test redirecting to the OrangeHRM website in a new tab."""
+    expected_url = "https://www.orangehrm.com/"
+    login_page.orangehrm_link.click()
+    assert login_page.switch_to_second_tab_and_check_url(expected_url), "Redirection was not performed"
```

Comment:
I broke test by changing expected_url. In this case we got 
`test_redirecting_to_orangehrm - selenium.common.exceptions.TimeoutException: Message:` not assertion "Redirection was not performed". So better navigate to second tab, get url and then compare it with expected